srv/GetCartesianPath.srv

```diff
@@ -43,6 +43,15 @@ bool avoid_collisions
 # Specify additional constraints to be met by the Cartesian path
 Constraints path_constraints
 
+# Scaling factors for optionally reducing the maximum joint velocities and
+# accelerations.  Allowed values are in (0,1].  The maximum joint velocity and
+# acceleration specified in the robot model are multiplied by thier respective
+# factors.  If either are outside their valid ranges (importantly, this
+# includes being set to 0.0), the factor is set to the default value of 1.0
+# internally (i.e., maximum joint velocity or maximum joint acceleration).
```

Comment:
```suggestion
# Scaling factors for optionally reducing the maximum joint velocities and
# accelerations.  Allowed values are in (0,1]. The maximum joint velocity and
# acceleration specified in the robot model are multiplied by their respective
# factors. If a factor is outside its valid range (importantly, this
# includes being set to 0.0), this factor is set to the default value of 1.0
# internally (i.e., maximum joint velocity or maximum joint acceleration).
```
Sorry about the nitpick, I thought this is a bit clearer (please correct me if I am wrong here!)

Reply:
I think the suggestions are great 👍🏼
I had copied the original comments from the corresponding fields in [MotionPlanRequest msg](https://github.com/ros-planning/moveit_msgs/blob/a511441601ae844678bdfcdde4c03f72877053d6/msg/MotionPlanRequest.msg#L45). Do you recommend pushing the same changes to that msg too or should we leave that for a separate PR?

Reply:
That would be nice, thank you!

Reply:
Done [a750605](https://github.com/ros-planning/moveit_msgs/pull/154/commits/a75060596e5efb34cb3e0d3ad37fb73a4e6488af) and [bbc01fc](https://github.com/ros-planning/moveit_msgs/pull/154/commits/bbc01fcbc08255fecd983d9e7500a599a6cf83e4)